src/main/kotlin/de/smartsquare/starter/mqtt/MqttProperties.kt

```diff
@@ -71,6 +72,15 @@ data class MqttProperties(
      * The shutdown configuration for the mqtt processor.
      */
     val shutdown: MqttShutdown = MqttShutdown.GRACEFUL,
+
+    /**
+     * The session expiry interval in seconds.
+     * An expiry of 4_294_967_295 means the session will never expire.
+     * This is also the max value and the default.
+     */
+    @get:Min(0)
+    @get:Max(Mqtt5Connect.NO_SESSION_EXPIRY)
+    val sessionExpiry: Long = Mqtt5Connect.NO_SESSION_EXPIRY,
```

Comment:
The mqtt client has `DEFAULT_SESSION_EXPIRY_INTERVAL` (0) as a default. I think it would be better to go with that, right?

Reply:
Yes, I think you're right. Thinking about it it makes even more sense from the client pov to keep the change as minimal invasive as possible.